lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -171,7 +185,12 @@ class CyberSourceGateway < Gateway
       # :ignore_cvv => true   don't want to use CVV so continue processing even
       #                       if CVV would have failed
       def initialize(options = {})
-        requires!(options, :login, :password)
+        if options[:public_key]
+          options[:merchant_id] ||= options[:login]
+          requires!(options, :merchant_id, :public_key, :private_key)
```

Comment:
Is there a reason you decided to change it from login to merchant_id?

Reply:
Only that merchant_id is more descriptive to the value it corresponds to, however, I'm going to remove it to only have login